pkg/cloud-provider/cloudapi/azure/azure_security.go

```diff
@@ -221,6 +234,7 @@ func (computeCfg *computeServiceConfig) processAddressGroupMembership(addressGro
 	return computeCfg.processAsgAttachDetachConcurrently(asgObj, nwIntfIDSetAsgToAttach, nwIntfIDSetAsgToDettach)
 }
 
+// processAsgAttachDetachConcurrently attaches/detaches network interfaces to/from an ASG.
```

Comment:
ASG or NSG?

Reply:
This is an ASG, we have a separate function for NSG(processNsgAttachDetachConcurrently) used for AT group membership update.